example/full-config.php

```diff
@@ -57,6 +57,10 @@
                     'regions' => [],
                 ],
                 'sql_logger' => null,
+                'middlewares' => [
+                    'app.foo.middleware', // Will be looked up in the container.
```

Comment:
Possibly worth adding a link to the documentation (in `doctrine/dbal`) for middleware :)